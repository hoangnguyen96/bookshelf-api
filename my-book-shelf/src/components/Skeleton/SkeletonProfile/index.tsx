import { Box, Flex, Skeleton } from "@chakra-ui/react";

const SkeletonProfile = () => (
  <Box p="5px 44px 20px">
    <Flex
      flexDir="column"
      bgColor="white"
      borderRadius="10px"
      width="100%"
      maxW={1136}
      p="48px 24px 24px"
      boxShadow="0 0 5px 1px rgb(0 0 0 / 10%)"
    >
      <Skeleton ml="9px" w="200px" h={25} borderRadius="10px" />
      <Flex
        flexDir="column"
        alignItems="center"
        alignSelf="flex-start"
        mt="50px"
      >
        <Skeleton ml="9px" w="300px" h={185} borderRadius="10px" mb="34px" />
      </Flex>
      <Flex alignItems="center" gap="40px" justifyContent="center" mt="50px">
        <Skeleton ml="9px" w="100%" h={45} borderRadius="10px" mb="34px" />
        <Skeleton ml="9px" w="100%" h={45} borderRadius="10px" mb="30px" />
      </Flex>
      <Flex alignItems="center" gap="40px" justifyContent="center">
        <Skeleton ml="9px" w="100%" h={45} borderRadius="10px" mb="34px" />
        <Skeleton ml="9px" w="100%" h={45} borderRadius="10px" mb="34px" />
      </Flex>
      <Skeleton ml="9px" w="100%" h={100} borderRadius="10px" mb="34px" />
      <Skeleton ml="9px" w="200px" h={50} borderRadius="10px" mb="34px" />
    </Flex>
  </Box>
);

export default SkeletonProfile;
