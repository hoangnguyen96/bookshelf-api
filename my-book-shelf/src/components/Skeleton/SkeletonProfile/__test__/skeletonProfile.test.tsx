import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import SkeletonProfile from "..";

describe("SkeletonProfile", () => {
  it("Should render correctly snapshot", () => {
    expect(render(<SkeletonProfile />)).toMatchSnapshot();
  });
});
