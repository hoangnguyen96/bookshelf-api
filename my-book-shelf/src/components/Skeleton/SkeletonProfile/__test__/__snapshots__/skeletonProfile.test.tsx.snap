// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonProfile Should render correctly snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-3sxpgv"
      >
        <div
          class="css-1at2akz"
        >
          <div
            class="chakra-skeleton css-2vhxyl"
          />
          <div
            class="css-19pzbu8"
          >
            <div
              class="chakra-skeleton css-auucw8"
            />
          </div>
          <div
            class="css-c8hlhc"
          >
            <div
              class="chakra-skeleton css-1twr6wk"
            />
            <div
              class="chakra-skeleton css-1ndnqtz"
            />
          </div>
          <div
            class="css-18n5hek"
          >
            <div
              class="chakra-skeleton css-1twr6wk"
            />
            <div
              class="chakra-skeleton css-1twr6wk"
            />
          </div>
          <div
            class="chakra-skeleton css-kgib2j"
          />
          <div
            class="chakra-skeleton css-1h1xlyp"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-3sxpgv"
    >
      <div
        class="css-1at2akz"
      >
        <div
          class="chakra-skeleton css-2vhxyl"
        />
        <div
          class="css-19pzbu8"
        >
          <div
            class="chakra-skeleton css-auucw8"
          />
        </div>
        <div
          class="css-c8hlhc"
        >
          <div
            class="chakra-skeleton css-1twr6wk"
          />
          <div
            class="chakra-skeleton css-1ndnqtz"
          />
        </div>
        <div
          class="css-18n5hek"
        >
          <div
            class="chakra-skeleton css-1twr6wk"
          />
          <div
            class="chakra-skeleton css-1twr6wk"
          />
        </div>
        <div
          class="chakra-skeleton css-kgib2j"
        />
        <div
          class="chakra-skeleton css-1h1xlyp"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
